tests/test_username_api.py

```diff
@@ -259,63 +267,63 @@ def test_available_twitter_username(self, user):
 
   def test_github_format_checking(self):
     resp = self.app.get('/check/github/{}'.format(invalid_username))
-    json_resp = json.loads(resp.get_data())
+    json_resp = json.loads(resp.get_data().decode())
```

Comment:
I don't know how tests were passing on travis in the first place. ``resp.get_data()`` returns bytes and had to be decoded. @andrewda any idea?

Reply:
Hmm it was working fine locally for me... Not sure

Reply:
This isnt a Python 2 vs 3 problem?

Is this app Python 2 compatible?  Anyway, maybe someone wants to investigate https://github.com/manu-chroma/username-availability-checker/issues/38 ;-)

Reply:
The problem is both these builds: [#6](https://travis-ci.org/manu-chroma/username-availability-checker/builds/318026284) and [#8](https://travis-ci.org/manu-chroma/username-availability-checker/builds/318231491) use python 3.6 and both of them pass.